### PR TITLE
Created Polygon Shape

### LINKED ISSUE #42: Polygon shape
We could include a general polygon shape where a sides=(3-10) prop would determine the number of sides. Based on that prop users would see
3-triangle
4-square
5-pentagon
...

### PR DESCRIPTION
**Pull request by JLambertazzo**
Closes #42 
Created polygon shape with clip-path depending on sides prop (sides in [3,10])

### CODE REVIEW

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
One minor change, Otherwise LGTM!

File: src/lib/shapes/polygon.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import styled from '@emotion/styled';
 import { BaseShape } from './base';
+import { clipPathMap } from './polygonclippath.tsx'
```
Comment (Sachin-chaurasiya) [inline src/lib/shapes/polygon.tsx:4]: The filename should be `polygonclippath.ts`.

**Comment by JLambertazzo** [comment]
Made requested changes also forgot to reference this is Issue #42

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM! , @ashutosh1919 Please review once.

**Review by ashutosh1919 — APPROVED** [review]
LGTM. Thanks for contribution @JLambertazzo .

**Pull request merged**